WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -573,6 +568,13 @@ public void onClick(View v) {
         });
     }
 
+    @Override
+    public void onClick(View view) {
+        if (view.getId() == R.id.row_register_domain) {
+            ActivityLauncher.viewDomainRegistrationActivity(this, getSelectedSite());
```

Comment:
Just to be consistent with the rest of the code, we should probably use inline `onClick` with `R.id.row_register_domain` too :)

Reply:
Good call. I needed to access the instance of `MySiteFragment` so I added `onClick` to the fragment class. This allowed me to use `this` to reference the fragment instance. I have made it an inline `onClick` as that is much more consistent with the result of the code. Thanks.